.github/workflows/build.yml

```diff
@@ -35,11 +38,37 @@ jobs:
           deploy-pull-request: true
           artifactory-reader-role: private-reader    # Override default public-reader
           artifactory-deployer-role: qa-deployer     # Override default public-deployer
+          sonar-platform: none
+          maven-args: -Dinvoker.skip=true
+
+  scan:
+    needs:
+      - quick-build-deploy
+    runs-on: github-ubuntu-latest-s  # Public repo uses custom GitHub-hosted runners
+    name: SQ Analysis
+    permissions: &read_permissions
+      id-token: write  # Required for Vault OIDC authentication
+      contents: read
+    env:
+      BUILD_NUMBER: ${{ needs.quick-build-deploy.outputs.build-number }}
+    steps:
+      - *checkout
+      - *mise
+      - name: Cache Sonar Scanner artifacts
+        id: sonar-scanner-cache
+        uses: SonarSource/ci-github-actions/cache@v1
+        with:
+          path: ~/.sonar/cache
+          key: sonar-scanner-${{ runner.os }}
+      - uses: SonarSource/ci-github-actions/build-maven@v1
+        with:
+          deploy: false
+          artifactory-reader-role: private-reader    # Override default public-reader
+          artifactory-deployer-role: qa-deployer     # Override default public-deployer
 
   invoker-integration-tests:
     needs:
-      - build
-    if: ${{ needs.build.outputs.deployed }}
```

Comment:
We had some trouble this week with the former `build` and `invoker-integration-tests` would be using different build numbers (and artifacts) producing errors on the pipeline.
Maybe we can keep the `if` guard for now